src/components/NftSnapshot.vue

```diff
@@ -220,12 +220,8 @@ export default {
       }
       return 0
     },
-    total_used () {
-      let value = 0
-      for (let item of this.stored) {
-        value = value + item.content.size
-      }
-      return value / (1024 ** 2)
+    total_used (state) {
```

Comment:
Should we add the unit here ?
```suggestion
    total_used_MiB (state) {
```